app/routes/events/view/edit/sessions-speakers.js

```diff
@@ -14,7 +14,19 @@ export default class SessionsSpeakersRoute extends Route.extend(EventWizardMixin
     const tracksPromise = data.event.get('tracks');
     const microlocationsPromise = data.event.get('microlocations');
     const sessionTypesPromise = data.event.get('sessionTypes');
-    const speakersCallPromise = this.getOrCreate(data.event, 'speakersCall', 'speakers-call');
+    const speakersCallPromise = new RSVP.Promise(resolve => {
+      data.event
+        .get('speakersCall')
+        .then(relationshipRecord => {
+          resolve(relationshipRecord);
```

Comment:
This then call is redundant

Reply:
after removing then, the session page isn't loading